### PR TITLE
[IMP] mail: make messaging and models available in models without env

### PR DESCRIPTION
**Pull request by seb-odoo**
Part of task-2582313

### CODE REVIEW

File: addons/mail/static/src/utils/test_utils.js [hunk 1]
```diff
@@ -399,7 +393,7 @@ async function createRootComponent(self, Component, { props = {}, target }) {
  *     @see addTimeControlToEnv that adds `advanceTime` function in
  *     `env.testUtils`.
  * @param {boolean} [param0.hasView=false] if set, use createView to create a
- *   view instead of a generic widget.
+ * @param {boolean} [param0.loadingBaseDelayDuration=0]
```
Comment (alexkuhn) [inline addons/mail/static/src/utils/test_utils.js:396]: integer, not boolean

File: addons/mail/static/src/model/model_manager.js [hunk 2]
```diff
@@ -260,6 +262,29 @@ export class ModelManager {
...
+    }
+
+    /**
+     * Returns all generated models.
+     *
+     * @returns {Object[]} keys are model name, values are model class
+     */
+    get models() {
+        return this._models;
+    }
```
Comment (alexkuhn) [inline addons/mail/static/src/model/model_manager.js:286]: I would prefer no getter and just return the attribute (and remove the _ prefix).
Looks straight-forward for `models`.
For `messaging`, the tracking of listening access is essential so I wouldn't mind keeping the getter approach for the time being.

**Review by alexkuhn — COMMENTED** [review]
Overall looks good

File: addons/mail/static/src/model/model_manager.js (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/src/model/model_manager.js:286]: ok

File: addons/mail/static/src/utils/test_utils.js (hunk 1 above)
Comment (seb-odoo) [reply addons/mail/static/src/utils/test_utils.js]: done

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Review by alexkuhn — COMMENTED** [review]
@robodoo rebase-ff

**Pull request closed without merging**